extensions/amp-slides/0.1/amp-slides.js

```diff
@@ -52,15 +52,16 @@ class AmpSlides extends AMP.BaseElement {
 
     this.prevButton_ = document.createElement('button');
     this.prevButton_.textContent = '\u276E';
-    this.prevButton_.style.opacity = 0.6;
-    this.prevButton_.style.position = 'absolute';
-    this.prevButton_.style.zIndex = 10;
-    this.prevButton_.style.left = '16px';
-    this.prevButton_.style.top = '50%';
-    this.prevButton_.style.padding = '8px';
-    this.prevButton_.style.fontSize = '24px';
-    this.prevButton_.style.marginTop = '-20px';
-    this.prevButton_.style.pointerEvents = 'all';
+    const prevButtonStyle = this.prevButton_.style;
```

Comment:
lets go ahead and use:

```js
st.setStyles(this.prevButton_, {
  //styles here...
});
```

it should already be imported in this file. Thanks!